docs/source/conf.py

```diff
@@ -103,6 +103,14 @@
 # TODO: verify this works as expected
 release = 'master'
 
+# Customized html_title here. 
+# Default is " ".join(project, release, "documentation") if not set
+if RELEASE:
+    html_title = " ".join((project, torch.__version__, "documentation"))    
+else:
```

Comment:
We should find some way to mark the master documentation as "not ready for general consumption". Maybe it would be better if the master documentation is still called "PyTorch master documentation" ?

Reply:
keep master as master, stable version will use 1.x.x. Remove the hash code (start with 'a') if any. eg. 1.5.0a4b+xxx -> 1.5.0